docs/source/evaluation_scripts.md

```diff
@@ -129,3 +129,205 @@ Here are the steps to configure remote evaluation:
+
+        ```python
+        class RandomAgent(habitat.Agent):
+            def __init__(self, task_config: habitat.Config):
+                self._POSSIBLE_ACTIONS = task_config.TASK.POSSIBLE_ACTIONS
+
+            def reset(self):
+                pass
+
+            def act(self, observations):
+                return {
+                    "action": ("ARM_ACTION", "BASE_VELOCITY", "REARRANGE_STOP"),
+                    "action_args": {
+                        "arm_action": np.random.rand(7),
+                        "grip_action": np.random.rand(1),
+                        "base_vel": np.random.rand(2),
+                        "REARRANGE_STOP": np.random.rand(1),
+                    },
+                }
+        ```
+
+        Note that this example is specific to superclass `habitat.Agent` and the corresponding environment is `habitat-lab`'s [`Challenge`](https://github.com/facebookresearch/habitat-lab/blob/b1f2d4791a0065d0791001b72a6c96748a5f9ae0/habitat-lab/habitat/core/challenge.py#L9) which makes the agent perform actions on the environment using the `reset` and `act` methods.
+
+        The `agent.py` should also contain a `main()` function which either submits the agent or the action to the environment.
+
+        We provide a template for `agent.py` [here](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/code_upload_challenge_evaluation/agent/agent.py):
+
+        ```python
+        import evaluation_pb2
+        import evaluation_pb2_grpc
+        import grpc
+        import os
+        import pickle
+        import time
+
+        time.sleep(30)
+
+        LOCAL_EVALUATION = os.environ.get("LOCAL_EVALUATION")
+
+        if LOCAL_EVALUATION:
+            channel = grpc.insecure_channel("environment:8085")
+        else:
+            channel = grpc.insecure_channel("localhost:8085")
+
+        stub = evaluation_pb2_grpc.EnvironmentStub(channel)
+
+        def pack_for_grpc(entity):
+            return pickle.dumps(entity)
+
+        def unpack_for_grpc(entity):
+            return pickle.loads(entity)
+
+        flag = None
+
+        while not flag:
+            base = unpack_for_grpc(
+                stub.act_on_environment(
+                    evaluation_pb2.Package(SerializedEntity=pack_for_grpc(1))
+                ).SerializedEntity
+            )
+            flag = base["feedback"][2]
+            print("Agent Feedback", base["feedback"])
+            print("*"* 100)
+
+        ```
+
+        This is a very basic example where the the agent sends action (`1`) to the environment and the environment sends back feedback after acting on the environment with the sent action.
+
+        The feedback is checked for stop signal, and usually contains observations from the sensors. The action then can depend on the `feedback` from the environment.
```

Comment:
Wait what is the difference between `RandomAgent` and this example? Shouldn't there be just 1 example? This seems confusing

Reply:
Removed `RandomAgent` and provided a link to it instead.